<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggingSettings</class>
 <widget class="QWidget" name="LoggingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logging settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="levelGroup">
     <property name="toolTip">
      <string>Control the log level of the internal logging of the Safir SDK Core.</string>
     </property>
     <property name="title">
      <string>Log Level</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="logLevelSlider">
        <property name="toolTip">
         <string>0 is no logging and 9 is very verbose logging.</string>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="logLevelSpin">
        <property name="toolTip">
         <string>0 is no logging and 9 is very verbose logging.</string>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroup">
     <property name="toolTip">
      <string>These options can be used to reduce the overhead of logging.</string>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="ignoreFlush">
        <property name="toolTip">
         <string>Ignore flushes, and write only when buffer is full..</string>
        </property>
        <property name="text">
         <string>Ignore Flush</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="timestamps">
        <property name="toolTip">
         <string>Put timestamps on each line in the logs.</string>
        </property>
        <property name="text">
         <string>Timestamps</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="toStdout">
        <property name="toolTip">
         <string>Log to standard output.</string>
        </property>
        <property name="text">
         <string>Log to stdout</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logLevelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>logLevelSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logLevelSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>logLevelSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
